src/daemon/absorber.cc

```diff
@@ -37,16 +35,17 @@ Absorber::~Absorber() {
 }
 
 bool Absorber::Initialize() {
+  auto conf = this->conf();
```

Comment:
Otherwise compiler gives an error about definition of `auto` variable. Also it's a little syntax trick to prevent implicit usage of raw `conf()`.